arrow-core-test/src/main/kotlin/arrow/core/Try.kt

```diff
@@ -141,3 +141,10 @@ fun <A> A.success(): Try<A> = Success(this)
 fun Throwable.failure(): Try<Nothing> = Failure(this)
 
 fun <T> TryOf<TryOf<T>>.flatten(): Try<T> = fix().flatMap(::identity)
+
+class ForTry private constructor() { companion object }
```

Comment:
shouldn't Try be removed already for 0.12? 🤔

Reply:
It's still needed by kotlintest

Reply:
Soon then :D